### DISCUSSION TITLE
Best installation method?

### DISCUSSION THREAD

**Discussion by mchn-hed** [regolith-linux/regolith-desktop#898] [Q&A]
So first off great job, super convenient to have one place that pulls together i3 and everything else you need to make it just work. I've been playing around over the weekend with it as well as reading over the forum so if I missed something or failed to understand I apologize in advance.

I've tried the standalone desktop 2.2 - which I believe is still considered as early beta - does it incorporate any of the fixes/changes from the 3.0 install that lays over ubuntu/debian? What would you recommend as the best experience (as in debian base+reg/ubuntu base+reg/regolith 2.2)? if ubuntu which version? Any recommendation on dealing with snaps? One of the the issues I had on 2.2 was the updater wanting to switch firefox into a snap package and finally is the link on the website the best place to donate? 

Thanks for the hard work and sharing it! If you're going to be active for the next 6 months to a year I'd like to invest the time into getting settled in

**Comment by kgilmer** [comment]
Hi @mchn-hed 

> I've tried the standalone desktop 2.2 - which I believe is still considered as early beta - does it incorporate any of the fixes/changes from the 3.0 install that lays over ubuntu/debian?

3.0 is the successor to 2.2.  3.0 was released a few weeks ago.  If you have an existing 2.2 install you can upgrade to 3.0 by changing the URL in the `apt` configuration for Regolith, as describe on the install page: https://regolith-desktop.com/docs/using-regolith/install/

> What would you recommend as the best experience

I guess it depends on your definition of "best".  Ubuntu is more commonly supported, but you may prefer Debian if you dislike Snaps or other things that Ubuntu brings in. 

Due to time required to develop and test, we only ship ISO distribution of Regolith (AKA "Regolith Linux") on LTS releases.  

You could choose either Ubuntu or Debian as your base.  If Debian, you must install the Regolith Desktop into an installed Debian system, as described on the Install page linked above.  If you choose Ubuntu, you can do the same or you can start with the Regolith 2.2 ISO and then upgrade the Ubuntu version to latest (Lunar), and then upgrade Regolith to latest (3.0).  

Overall I think it would be simplest to start with Ubuntu Lunar or Debian Bookworm and install Regolith Desktop into it.

Regarding  Firefox/Snap, there is information available via a google search to replace the snap version with debs provided by upstream.  Here's how it's done in the 2.2 ISO: https://github.com/regolith-linux/regolith-ubuntu-iso-builder/blob/main/scripts/regolith-2_2-kinetic-config.sh#L72-L77

> is the link on the website the best place to donate?

Yes and thank you in advance for any contribution you make!
